eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -96,6 +96,7 @@ jobs:
     CMOCKA_XML_FILE: "%g-test-results.xml"
     CMOCKA_MESSAGE_OUTPUT: "xml"
     AZURE_ENABLE_STATIC_ANALYSIS: 1
+    AZURE_KEYVAULT_AVOID_THROTTLED: 1
```

Comment:
If this is specific to keyvault does it make more sense to add it to the ci.yml file for keyvault? Assuming we a way to plumb that through if not we might want to start thinking about a way to plumb it through to avoid service specific settings going directly into the shared template.

Reply:
I know how to set args from the ci.yml file to the archtype-sdk-tests.yml, but I don't know how to set env variables directly from the ci.yml.  Would it be just adding it at the end?